cypcore/Consensus/Blockmania.cs

```diff
@@ -762,13 +762,7 @@ private async Task Run(ChannelReader<BlockGraph> reader)
         public void Add(BlockGraph data)
         {
             _logger.Here().Debug("Adding block to graph block.id: {@BlockId}", data.Block);
-
-            var task = Task.Factory.StartNew(async () =>
-            {
-                await _entries.Writer.WriteAsync(data);
-            });
-
-            task.Wait();
+            Task.Factory.StartNew(async () => { await _entries.Writer.WriteAsync(data); });
```

Comment:
So this thread runs forever and cannot be gracefully cancelled?

Reply:
Same as with a blocking thread no?